domain-management/src/main/java/org/jboss/as/domain/management/security/adduser/DisplaySecret.java

```diff
@@ -43,7 +45,7 @@ public DisplaySecret(ConsoleWrapper theConsole, final StateValues stateValues) {
     }
 
     public State execute() {
-        String pwdBase64 = Base64.getEncoder().encodeToString(stateValues.getPassword().getBytes());
+        String pwdBase64 = Base64.getEncoder().encodeToString(stateValues.getPassword().getBytes(StandardCharsets.UTF_8));
```

Comment:
Same as the decode this by default uses ISO_8859_1. Again though I don't think it's a big deal.